modules/channel-web/src/views/lite/components/Composer.tsx

```diff
@@ -58,7 +58,10 @@ class Composer extends React.Component<ComposerProps> {
   render() {
     const placeholder =
       this.props.composerPlaceholder ||
-      this.props.intl.formatMessage({ id: 'composer.placeholder' }, { name: this.props.botName })
+      this.props.intl.formatMessage({
+        id: this.props.currentConversation &&
+            this.props.currentConversation.messages &&
+            this.props.currentConversation.messages.length > 1 ? 'composer.placeholder' : 'composer.placeholderInit' }, { name: this.props.botName })
```

Comment:
@SaidMarar I'm Pretty sure you meant >=  

```suggestion
            this.props.currentConversation.messages.length >= 1 ? 'composer.placeholder' : 'composer.placeholderInit' }, { name: this.props.botName })
```

Reply:
@EFF Thak you for your review, actually there is a scenario where the last message is not from bot in that case we should put "Say something to" please check screenshots and the new code.

1. ### empty conversation:
![image](https://user-images.githubusercontent.com/20561190/98384831-8d97b600-204e-11eb-962f-4374693ed331.png)

2. ### only user:
![image](https://user-images.githubusercontent.com/20561190/98384874-9b4d3b80-204e-11eb-9397-9833b5bf1dd5.png)

3. ### bot replied:
![image](https://user-images.githubusercontent.com/20561190/98384885-a2744980-204e-11eb-937c-9ebc11918074.png)

4. ### full conversation:
![image](https://user-images.githubusercontent.com/20561190/98384910-a99b5780-204e-11eb-8841-167037493f4f.png)

cc: @allardy 
**Merci**